source/jquery.shinybox.js

```diff
@@ -202,13 +202,8 @@
 
                 if ("onorientationchange" in window) {
                     var calculateWidthAndHeight = function () {
-                        if (window.orientation === 0) {
-                            width = winWidth;
-                            height = winHeight;
-                        } else if (window.orientation === 90 || window.orientation === -90) {
-                            width = winHeight;
```

Comment:
Hey @sathishkategaru, here you no longer react to the orientation of 90 or -90 degrees by using the winHeight as width, and winWidth as height, which I think will cause issues when you rotate the device.
From the PR description I can't see if you tested device orientation or not, could you check that? If that does still work, could you explain exactly _how_ it calculates the correct width/height when the device is rotated?